File: app/src/main/java/com/example/biblio/helpers/ThemeHelper.java
package com.example.biblio.helpers;

import androidx.appcompat.app.AppCompatDelegate;

public class ThemeHelper {
    public static void applyTheme(boolean themeOption) {
        if (themeOption)
            AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
        else
            AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
    }
}

File: app/src/main/java/com/example/biblio/helpers/ThemeHelper.kt
package com.example.biblio.helpers

import androidx.appcompat.app.AppCompatDelegate

object ThemeHelper {
    @JvmStatic
    fun applyTheme(themeOption: Boolean) {
        AppCompatDelegate.setDefaultNightMode(when (themeOption) {
            true -> AppCompatDelegate.MODE_NIGHT_YES
            false -> AppCompatDelegate.MODE_NIGHT_NO
        })
    }
}